sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -1188,6 +1188,24 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
       }
     }
   }
+
+  test("SPARK-28054: Unable to insert partitioned table when partition name is upper case") {
+    withTable("spark_28054_test") {
+      sql("set hive.exec.dynamic.partition.mode=nonstrict")
```

Comment:
Should we set it back?

Use withSQLConf ?

Also do we need to set the case sensitivity conf?

Reply:
This set follows other tests in same suite. Using withSQLConf is good, yes.

The case sensitivity conf has no effect on this, I think it is fine.